scripts/dev/verify_idfs_in_cmake.py

```diff
@@ -94,7 +94,7 @@
 # there are a few files we purposely skip
 files_to_skip = {"_1a-Long0.0.idf", "_ExternalInterface-actuator.idf", "_ExternalInterface-schedule.idf",
                  "_ExternalInterface-variable.idf", "HVAC3Zone-IntGains-Def.imf", "HVAC3ZoneChillerSpec.imf",
-                 "HVAC3ZoneGeometry.imf", "HVAC3ZoneMat-Const.imf"}
+                 "HVAC3ZoneGeometry.imf", "HVAC3ZoneMat-Const.imf", "_1ZoneUncontrolled_ForAPITesting.idf"}
```

Comment:
I made a special IDF that I can fiddle with all willy-nilly and not bother the core 1ZoneUncontrolled.idf file.  But I need to ignore it here in custom_check.